src/SqlAsyncCollector.cs

```diff
@@ -171,6 +172,16 @@ private async Task UpsertRowsAsync(IEnumerable<T> rows, SqlAttribute attribute,
                 ObjectCache cachedTables = MemoryCache.Default;
                 var tableInfo = cachedTables[cacheKey] as TableInformation;
 
+                int timeout = 10;
```

Comment:
Make a `DEFAULT_TABLE_INFO_CACHE_TIMEOUT_MIN` var and set this to that initially.